query-string-parser.js

```diff
@@ -111,103 +104,9 @@ function removeEmptyParams (query) {
   return pruned
 }
 
-function parseAgentScope (query, data) {
```

Comment:
In the `parse()` function above, is a comment from me " Why did I think we needed this???".  Duplicated.  Heh.  We need exactly 0 copies of this block comment.

Right under that is some checking for `interval` and `agg`, which can be removed.